frontend/src/components/Auth/SignUp/SignUp.jsx

```diff
@@ -203,7 +203,7 @@ const SignUp = () => {
         </Grid>
         <Grid container justifyContent="center">
           <Grid item>
-            <Link href="#" variant="body2" color="secondary">
+            <Link href="/login" variant="body2" color="secondary">
```

Comment:
We do have it in constant variable in paths 
Change it to PATH_TO_LOGIN

Reply:
done